Assets/Global/Scripts/Player/Player.cs

```diff
@@ -217,8 +224,8 @@ private void ApproachTargetVelocity() {
     public void OnHit(float damage)
     {
         playerStatistic.Health -= damage;
+        GlobalReference.PermanentPlayerStatistic.Health = playerStatistic.Health;
```

Comment:
This is a sideeffect of the fact that you have Health in the permanenet stats i think.

So this will be removed because that is being removed anyways.
But in theory we should never have to refer to permanent stats in other parts of the code other than the bits and bos that are Directly related to it. 
Aka. OvenOfEternity. Aka. CurrentStats.